packages/core/test/rehydration.test.ts

```diff
@@ -73,9 +73,11 @@ describe('rehydration', () => {
         }
       });
 
-      const service = interpret(machine);
+      const activeState = machine.transition(undefined, { type: 'NEXT' });
```

Comment:
Shouldn't this be rewritten like this?
```suggestion
      const activeState = machine.resolveStateValue('active');
```